CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## v0.44.1 (April 21, 2023)
 
+FEATURES:
+* `r/tfe_organization_token`: Add optional `expired_at` field to organization tokens by, @juliannatetreault ([#844](https://github.com/hashicorp/terraform-provider-tfe/pull/844))
```

Comment:
Not urgent (and probably not worth doing until we are actually ready to release this), but a heads up that we will need to change where in the changelog this is, up to the "Unreleased" section